src/profile.c

```diff
@@ -83,6 +82,11 @@ static double _recursiveProfilePrint(RedisModule_Reply *reply, ResultProcessor *
       case RP_MAX:
         RS_LOG_ASSERT(0, "RPType error");
         break;
+      case RP_SAFE_LOADER:
+        printProfileType(RPTypeToString(rp->type));
+        double rpGILTime = (double)RPProfile_GetGILTime(rp) / CLOCKS_PER_MILLISEC;
+        printProfileGILTime(rpGILTime);
+        break;
```

Comment:
1. Let's put this case above the `RP_PROFILE` and `RP_MAX` handling
2. Why doesn't any test cover this case?